src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -526,7 +558,10 @@ private void checkAndUpdateGains() {
         if (steerPositionPIDGains.hasChanged()) {
             SlotConfiguration newSlotConfig = new SlotConfiguration();
             steerPositionPIDGains.setSlot(newSlotConfig);
-            steerMotor.configureSlot(newSlotConfig);
+            steerController = steerMotor.getPIDController();
```

Comment:
I don't think you need to do this as it is already the correct reference

Reply:
nuh uh